bin/json_schema.py

```diff
@@ -182,6 +191,22 @@ def _handle_parameter_selection(self, d: dict, param_index: int) -> dict:
                     tmp_param_dict[param_name] = param_value[param_index]
                 return {"name": key, "params": tmp_param_dict}  
                 
+    
+    def unique_dicts_in_list(self, dict_list: list) -> list:
+        """
+        function is pretty straight forwrd: it checks if all elements in a list are unique and returns only unique once.
+        This is not a private function because is called from outside this file as well.
+        """
+        unique_list = []
+        for d in dict_list:
+            is_unique = True
+            for unique in unique_list:
+                if d == unique:
+                    is_unique = False
+                    break
+            if is_unique:
+                unique_list.append(d)
+        return unique_list
```

Comment:
Also why don't you just use `list(set(dict_list))`? It should do the same

Reply:
i tried a toy eaxample and to me that does not work:
````
>>> l = [{"a":1}, {"b":1}, {"a":1}]
>>> list(set(l))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'dict'